src/commonMain/kotlin/it/krzeminski/snakeyaml/engine/kmp/api/LoadSettingsBuilder.kt

```diff
@@ -49,7 +49,7 @@ class LoadSettingsBuilder internal constructor() {
     private var useMarks: Boolean = true
     private var envConfig: EnvConfig? = null // no ENV substitution by default
     private var codePointLimit: Int = 3 * 1024 * 1024 // 3 MB
-    private var schema: Schema = JsonSchema()
+    private var schema: Schema = CoreSchema()
```

Comment:
(A general thread for this PR - creating attached to a line of code to be able to respond in the thread - GitHub's inconsistency...)

As far as my understanding about the schemas go, I agree that we should go with changing the default. What I would push for, though, is having this default schema defined ideally in a single place. I think such a refactoring should go first, and then this functional PR would ideally be a one-liner. This way we ensure proper consistency as well (as long as the refactoring PR is properly reviewed).